docker-nersc/Dockerfile

```diff
@@ -209,6 +208,9 @@ RUN python -O -m compileall \
     /src/legacypipe/py/{legacypipe,legacyzpts}
 
 # update legacypipe
-# RUN cd /src/legacypipe && git pull && git checkout calib-container && git describe && echo 4 &&\
-#   python    -m compileall /src/legacypipe/py/{legacypipe,legacyzpts} && \
-#   python -O -m compileall /src/legacypipe/py/{legacypipe,legacyzpts}
+RUN cd /src/legacypipe && git pull && git checkout preburn-lslga && git describe && echo 1 && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use WORKDIR to switch to a directory](https://app.codacy.com/manual/dstndstn/legacypipe/pullRequest?prid=4930373)